fiftyone/utils/coco.py

```diff
@@ -511,8 +511,10 @@ def label_cls(self):
         return fol.Detections
 
     def setup(self):
-        self._image_id = -1
-        self._anno_id = -1
+        self._data_dir = os.path.join(self.export_dir, "data")
```

Comment:
I believe `_data_dir` and `_labels_path` are leftover from a merge conflict and need to be removed